core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -446,6 +446,12 @@ public static String textDistributedPlan(
         TypeProvider typeProvider = getTypeProvider(allFragments);
 
         builder.append(format("Trino version: %s\n", version));
+        builder.append(format("Planned: %s, Queued: %s, Scheduled: %s, Blocked: %s, Executed: %s\n",
```

Comment:
Let's add analysis time and finishing time and drop scheduled and blocked as they are easy to track from existing output.
Ordering and naming should be :
`Queued, Analysis, Planning, Execution, Finishing`

Reply:
Finishing time will be zero during generating `PlanPrinter `output, as query is not finished yet.